### ISSUE TITLE
Character Sheet/Character Skills - Skills/Skill Queue are not updated

### ISSUE THREAD

**Issue by MrBMT** [eveseat/seat#295]
I've not been able to work out where this issue is actually coming from, as I've not been able to find any failed jobs related to this, also based on the Last Update time provided, it seems like it should be getting updated.

I've done a clean install of SeAT a few times, and have always been able to replicate this.

* **Problem**: _The Skill Queue on the Character Sheet does not update. After a skill finishes training, it simply says Skill Training End (number) minutes etc. ago. The Skill Queue also does not get updated, so it does not move on to the next skill as the current one in training, or update the end time.
Edit: Having checked, the Character Skills page is also not being updated either._
* **Expected**: _Skill Queue updates._
* **Logs / Screenshots / Proof**: _See Below_
* **Version Info**: _PHP 7.1, Debian 9_
```
+---------------+---------------+
| Package Name  | Local Version |
+---------------+---------------+
| Api           | 3.0.0-beta5   |
| Console       | 3.0.0-beta3   |
| Eveapi        | 3.0.0-beta8   |
| Notifications | 3.0.0-beta2   |
| Web           | 3.0.0-beta7   |
| Services      | 3.0.0-beta6   |
+---------------+---------------+
```

![charactersheet](https://user-images.githubusercontent.com/6931281/39846402-6bd120a8-53f3-11e8-8a0f-7494186e6f31.png)

**Comment by herpaderpaldent** [comment]
TL;DR: Not a bug

Can not really confirm since my characters are on an aweful long train :-) 
However i think i know what's happening:
https://github.com/eveseat/web/blob/master/src/resources/views/character/sheet.blade.php#L32-L34

It only shows the first of the received data. Since the data is only updated upon a fresh login *or* if the scheduler hits the update (1 per hrs) this will update.
BUT: My latest contribution to the extractors handled with finish-dates as well, i don't think it is nearly as complex as the extractors. I'll see if i can fix it today

**Comment by MrBMT** [comment]
> It only shows the first of the received data. Since the data is only updated upon a fresh login or if the scheduler hits the update (1 per hrs) this will update.

Nope, that's not the case. I recently re-installed SeAT so this was the skill that finished soonest on one of my new characters, however it definitely does not update once an hour or if you re-login.

Here's the same thing 8 hours later:

![skillsheet](https://user-images.githubusercontent.com/6931281/39860116-0e3d6672-5434-11e8-888b-f41d88020b83.png)

**Comment by leonjza** [comment]
Should be fixed in latest web beta package.

**Comment by H4Z4RD-H42** [comment]
I still have this issue.

![image](https://user-images.githubusercontent.com/6650832/41165549-3bc96c96-6b3e-11e8-82e6-62703991f4bf.png)

I spoke with Zari Tenjin in Slack support channel about this and provide logs, This is the link of the discussion -> https://eve-seat.slack.com/archives/C0H2VM324/p1528327668000167

This is the log from horizon related to `Skill` job (`Skill.Queue` job have the same error) -> https://pastebin.com/BD0gPkZx

This happen with Seat 3.0.0beta (docker version) and Web version 3.0.0 beta20.

**Comment by warlof** [comment]
Is it me, or this instance has been esi denied? 

Can you turn on eseye debugging, track for skill endpoint and return us the result as soon as you have the failed job appear? 

Also, jobs are part of eveapi package, can you provide its version as well? (you'll get it from configuration)

**Comment by H4Z4RD-H42** [comment]
I'm sorry but not being an expert, it's very complicated to do what you ask. I tried yesterday and today for hours, with the help of Google and the Doc section of seat, but I do not think I found the right logs... You warlof saw it on slack.

**Comment by warlof** [comment]
all good, just wait for hotfix ;)

**Comment by H4Z4RD-H42** [comment]
Now skills queue it's ok, but there is still a bug on completition time:

![image](https://user-images.githubusercontent.com/6650832/41496922-120080d0-714b-11e8-9542-d836df872cb9.png)

![image](https://user-images.githubusercontent.com/6650832/41496934-5eb83224-714b-11e8-9754-f601671e4dd2.png)

**Comment by warlof** [comment]
@H4Z4RD-H42 I don't have the time of your in-game screen, but assuming that both the displayed date and time in SeAT are correct. The wrong thing is the human conversion (7 hours ago) ?

**Comment by H4Z4RD-H42** [comment]
Yes, i wrote that all it's ok except "Skill Training End" (borded in red in the screenshot).
Offcourse i took both screenshot at the same time when i wrote the post.
